editor/src/core/model/element-metadata-utils.ts

```diff
@@ -175,6 +176,8 @@ export const getChildrenOfCollapsedViews = (
   }, collapsedViews)
 }
 
+// TODO should this also be changed to simply TextElements?
+// (to include all possible text elements in the name lookup)
```

Comment:
Good question. This is used to determine what to show in the Navigator, and is only used if the element has _only_ text as a child, so I'd say this should include all of `TextElements` plus `div` at the very least. @maltenuhn Should we be using all of `PossibleTextElements` here?

Reply:
We can also do `['div', 'span'].concat(TextElements)`